.github/workflows/deploy_staging.yml

```
@@ -0,0 +1,43 @@
+name: Deploy to Staging
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build_and_push_image:
+    name: Build and Push Image
+    uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@optional-build-args
```

i think this will need an @main tag then..
```suggestion
    uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@main
```